### PR TITLE
Test coverage

### PR DESCRIPTION
**Pull request by alfonsomartinezs**

### CODE REVIEW

File: src/App.test.js [hunk 1]
```diff
@@ -1,8 +1,42 @@
...
+Enzyme.configure({ adapter: new Adapter() });
+
+describe("when app renders", () => {
+  it("renders a header and footer component", () => {
+    //arrange
+    const renderedApp = shallow(<App />);
+    //act
+    const renderedHeader = renderedApp.find("Header");
+    const renderedFooter = renderedApp.find("Footer");
+    // console.log("rendered header: ", renderedHeader);
```
Comment (sjproctor) [inline src/App.test.js]: There are a lot of console.logs left in here. Can you clean them up?

File: src/App.test.js [hunk 2]
```diff
@@ -1,8 +1,42 @@
...
+  it("provides a route to home component", () => {
+    //arrange
+    const renderedApp = shallow(<App />);
+    //act
+    const renderedHomeRoute = renderedApp.find('[path="/"]');
+    console.log("renderedHomeRoute debug: ", renderedHomeRoute.debug());
+    console.log("rendered Props: ", renderedHomeRoute.props());
+    console.log("home component: ", Home);
+
+    // const renderedNotFoundRoute = renderedApp.find(
```
Comment (sjproctor) [inline src/App.test.js]: If this is not being used, it can be removed.

File: src/pages/__tests__/NotFound.test.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  it("displays an image", () => {
+    //arrange
+    const notFound = shallow(<NotFound />);
+    //act
+    const notFoundImage = notFound.find("img");
+    console.log("image debug", notFoundImage.debug());
+    //assert
+    expect(notFoundImage.prop("src")).toEqual(cat404);
+    expect(notFoundImage.props().src).toEqual(cat404);
+  });
```
Comment (sjproctor) [inline src/pages/__tests__/NotFound.test.js:19]: Are both of these expects passing?

**Review by sjproctor — CHANGES_REQUESTED** [review]
Couple things to look at in the comments.

File: src/App.test.js (hunk 1 above)
Comment (alfonsomartinezs) [reply src/App.test.js]: cleaned up!

File: src/App.test.js (hunk 2 above)
Comment (alfonsomartinezs) [reply src/App.test.js]: thank you!

File: src/pages/__tests__/NotFound.test.js (hunk 3 above)
Comment (alfonsomartinezs) [reply src/pages/__tests__/NotFound.test.js:19]: yes, we were testing two different methods. left a comment to explain

**Review by sjproctor — APPROVED** [review]
Great!!

**Pull request merged**